Sources/SwiftDocC/Semantics/General Purpose Analyses/HasExactlyOne.swift

```diff
@@ -300,4 +300,94 @@ extension Semantic.Analyses {
             )
         }
     }
+    
+    public struct HasExactlyOneLink<Parent: Semantic & DirectiveConvertible>: SemanticAnalysis {
```

Comment:
This entire function wouldn't be needed if the link was passed as a parameter instead of as inner content.